Source/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 
+include_directories_exported(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
I noticed that, but since comment said it is for QtCreator i figured maybe it is just some dumb include to make IDE happy therefore i did not touch it. If someone could verify if IDE autocompletion without those lines still works it would be great. For some reason QtCreator crashes on my system. :|